INSTALL.md

```diff
@@ -136,6 +249,40 @@ It might be necessary to restart postgresql: `sudo service postgresql restart`.
 
 Furthermore, the `pg_hba.conf` file can be modified to change authentication type for local users to trust (rather than, e.g., md5) but we haven't tested this solution.
 
+### `Error: You need to install postgresql-server-dev-X.Y for building a server-side extension or libpq-dev for building a client-side application.`
```

Comment:
All these "troubleshooting" on Ubuntu. It looks like is a result of incomplete instructions rather than actual problems.